{"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"UnknownUtxo","kind":"identifier"},{"kind":"text","text":"("},{"text":"utxo","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"TxOut","kind":"typeIdentifier","preciseIdentifier":"s:13TapyrusWallet5TxOutV","identifier":"doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet\/TxOut"},{"text":")","kind":"text"}],"languages":["swift"]}],"kind":"declarations"}],"abstract":[{"type":"text","text":"Occur if the UTXO which means TxOut is not in the wallet db because_description the wallet is not synced or any other reasons"}],"variants":[{"paths":["\/documentation\/tapyruswallet\/transfererror\/unknownutxo(utxo:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"UnknownUtxo"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"utxo"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:13TapyrusWallet5TxOutV","text":"TxOut"},{"kind":"text","text":")"}],"modules":[{"name":"TapyrusWallet"}],"symbolKind":"case","role":"symbol","externalID":"s:13TapyrusWallet13TransferErrorO11UnknownUtxoyAcA5TxOutV_tcACmF","title":"TransferError.UnknownUtxo(utxo:)"},"hierarchy":{"paths":[["doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet","doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet\/TransferError"]]},"kind":"symbol","sections":[],"identifier":{"url":"doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet\/TransferError\/UnknownUtxo(utxo:)","interfaceLanguage":"swift"},"schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://TapyrusWallet/documentation/TapyrusWallet/TransferError":{"fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"TransferError"}],"type":"topic","abstract":[{"type":"text","text":"The error for HDWallet#transfer"}],"title":"TransferError","kind":"symbol","url":"\/documentation\/tapyruswallet\/transfererror","identifier":"doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet\/TransferError","navigatorTitle":[{"text":"TransferError","kind":"identifier"}],"role":"symbol"},"doc://TapyrusWallet/documentation/TapyrusWallet/TxOut":{"kind":"symbol","navigatorTitle":[{"text":"TxOut","kind":"identifier"}],"abstract":[{"type":"text","text":"The transaction output"}],"url":"\/documentation\/tapyruswallet\/txout","title":"TxOut","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"text":"TxOut","kind":"identifier"}],"type":"topic","role":"symbol","identifier":"doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet\/TxOut"},"doc://TapyrusWallet/documentation/TapyrusWallet":{"type":"topic","abstract":[],"title":"TapyrusWallet","kind":"symbol","url":"\/documentation\/tapyruswallet","identifier":"doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet","role":"collection"},"doc://TapyrusWallet/documentation/TapyrusWallet/TransferError/UnknownUtxo(utxo:)":{"abstract":[{"type":"text","text":"Occur if the UTXO which means TxOut is not in the wallet db because_description the wallet is not synced or any other reasons"}],"fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"UnknownUtxo"},{"kind":"text","text":"("},{"text":"utxo","kind":"externalParam"},{"kind":"text","text":": "},{"text":"TxOut","preciseIdentifier":"s:13TapyrusWallet5TxOutV","kind":"typeIdentifier"},{"kind":"text","text":")"}],"type":"topic","kind":"symbol","role":"symbol","identifier":"doc:\/\/TapyrusWallet\/documentation\/TapyrusWallet\/TransferError\/UnknownUtxo(utxo:)","url":"\/documentation\/tapyruswallet\/transfererror\/unknownutxo(utxo:)","title":"TransferError.UnknownUtxo(utxo:)"}}}